File: src/main/java/ru/mdashlw/hypixel/reply/impl/SessionReply.java
package ru.mdashlw.hypixel.reply.impl;

import com.fasterxml.jackson.annotation.JsonProperty;
import ru.mdashlw.hypixel.entities.Session;
import ru.mdashlw.hypixel.reply.Reply;

public class SessionReply extends Reply<Session> {
    @JsonProperty
    private Session session;

    @Override
    public Session getValue() {
        return session;
    }
}

File: src/main/kotlin/ru/mdashlw/hypixel/reply/impl/SessionReply.kt
package ru.mdashlw.hypixel.reply.impl

import ru.mdashlw.hypixel.elements.Session
import ru.mdashlw.hypixel.reply.Reply

class SessionReply(
    success: Boolean,
    cause: String?,
    throttle: Boolean,
    session: Session?
) : Reply<Session>(success, cause, throttle, session)
